public/src/js/models/article/editor.js

```diff
@@ -169,17 +169,18 @@ export default class Editor extends BaseClass {
         var imageSrc = this.article.meta[params.src],
             imageSrcWidth = this.article.meta[params.width],
             imageSrcHeight = this.article.meta[params.height],
-            imageSrcOrigin = this.article.meta[params.origin];
+            imageSrcOrigin = this.article.meta[params.origin],
+            imageSrcThumb = this.article.meta[params.imageSrcThumb] || function(){return '';};
```

Comment:
It's expected to be a Knockout observable I believe, so it's called to get a value